src/main/java/seedu/address/model/session/Session.java

```diff
@@ -40,4 +42,24 @@ public Subject getSubject() {
     public Fee getFee() {
         return fee;
     }
+
+    @Override
+    public int hashCode() {
+        // use this method for custom fields hashing instead of implementing your own
+        return Objects.hash(sessionDate, duration, subject, fee);
+    }
```

Comment:
What is this hashCode() function for?

Reply:
Possibly for use in the future to differentiate session objects, since we do not assign unique ids to sessions yet. If this is not needed I can remove it!

Reply:
Oh ok, let's leave it then!